### ISSUE TITLE
Remote directory to Workspace [*DIR]

### ISSUE THREAD

**Issue by juanpabloruiz** [humy2833/FTP-Simple#262]
Cuando me conecto directo al servidor por ftp para editar algún archivo, algunas carpetas me figuran con [*DIR] y si entro me las borra, que puede ser?

**Comment by GerardMetal** [comment]
The folders still download

**Comment by EnP-git** [comment]
Do you guys fixed that? how can I control that directory? still I have a problem to use because of this.

**Comment by GerardMetal** [comment]
> Do you guys fixed that? how can I control that directory? still I have a problem to use because of this.

Nope, my solution was to work with Git

**Comment by juanpabloruiz** [comment]
That's right, I also switched to GIT to be able to work on repositories and then push on the server

**Comment by abhimanyunaikare** [comment]
> > Do you guys fixed that? how can I control that directory? still I have a problem to use because of this.
> 
> Nope, my solution was to work with Git

So you are saying you stopped using ftp-simple. And switched to using git. Right?

**Comment by GMButters** [comment]
> 

Yep, ftp-simple is very buggy with medium and big projects

**Comment by mahesame** [comment]
Based on my experience, if you see [DIR] before the name of a directory in FTP Simple, it means that it's still downloading. Just wait for a moment, the [DIR] will disappear and we can access the directory.
